editor/block-settings-menu/content.js

```diff
@@ -9,14 +9,15 @@ import { flow } from 'lodash';
  */
 import { __ } from '@wordpress/i18n';
 import { IconButton } from '@wordpress/components';
+import { getBlockType } from '@wordpress/blocks';
 
 /**
  * Internal dependencies
  */
-import { isEditorSidebarOpened, getBlockMode } from '../selectors';
+import { isEditorSidebarOpened, getBlockMode, getBlock } from '../selectors';
 import { removeBlocks, toggleSidebar, setActivePanel, toggleBlockMode } from '../actions';
 
-function BlockSettingsMenuContent( { mode, uids, isSidebarOpened, onDelete, onToggleSidebar, onShowInspector, onToggleMode, onClose } ) {
+export function BlockSettingsMenuContent( { blockType, mode, uids, isSidebarOpened, onDelete, onToggleSidebar, onShowInspector, onToggleMode, onClose } ) {
```

Comment:
Should we consider splitting props onto their own lines, since this one is now quite long? Alternatively, would it be sensible to create a separate component solely responsible for deciding whether and how an "Edit as HTML" button should be rendered?

Reply:
Sounds like a good idae